kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -16,6 +18,12 @@ ChannelRouter = Backbone.Router.extend({
         this.navigate(this.default_channel + "/", {trigger: true, replace: true});
     },
 
+    intercept_learn_nav: function(event){
+        event.preventDefault();
```

Comment:
[According to the source](http://stackoverflow.com/a/1357151), `return`ing `false` will also already do `preventDefault`.